prow/cmd/deck/main_test.go

```diff
@@ -425,7 +425,7 @@ func TestRerun(t *testing.T) {
 			session.Values["access-token"] = &oauth2.Token{AccessToken: "validtoken"}
 
 			rr := httptest.NewRecorder()
-			mockConfig := &config.GitHubOAuthConfig{
+			mockConfig := &githuboauth.Config{
```

Comment:
I still don't really understand why we needed to move this package? Could we break these changes out to make the rest easier to review?

Reply:
So here is the thing, we want to be able to initialize oauth client from `prow/flagutil` package. There would be cyclic imports if we don't move it into a proper package.